components/container/index.test.jsx

```diff
@@ -78,6 +78,11 @@ describe("Container view", () => {
   const resourceIri = "https://example.com/container/resource.txt";
   const container = mockContainer(iri);
   const { dataset } = container;
+  const accessControlData = {
+    accessControl: mockAccessControl(),
+    isAcp: true,
+    isWac: false,
```

Comment:
You could collapse these 2 boolean values to one value that can represent the _type_ instead of hard-coding each boolean value of which type it is. I would change this to something like `accessControlType` with possible values of `"none"`, `"acp"`, or `"wac"`. This way, if a new access control type is introduced, you don't need to add another boolean value and push it all through the system where these booleans get passed.